native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -101,6 +104,33 @@ static int enable_options = -1; /* Use OPTIONS * for CPING/CPONG */
 #define TIMESESSIONID 300                    /* after 5 minutes the sessionid have probably timeout */
 #define TIMEDOMAIN    300                    /* after 5 minutes the sessionid have probably timeout */
 
+/* From apr_reslist.c
```

Comment:
Declaration is not enough :(, we need the definition for our check on ntotal, nidle...